packages/compass-sidebar/src/components/multiple-connections/active-connections/active-connection-navigation.tsx

```diff
@@ -49,16 +54,17 @@ interface Match {
   isMatch?: boolean;
 }
 
-type Collection = Connection['databases'][number]['collections'][number];
+type Collection =
+  ConnectedConnection['databases'][number]['collections'][number];
 
-type Database = Connection['databases'][number];
+type Database = ConnectedConnection['databases'][number];
 
 type FilteredCollection = Collection & Match;
 type FilteredDatabase = Omit<Database, 'collections'> &
   Match & {
     collections: FilteredCollection[];
   };
-type FilteredConnection = Omit<Connection, 'databases'> &
+type FilteredConnection = Omit<ConnectedConnection, 'databases'> &
```

Comment:
is this some preparation for the unified list? for colls and dbs it makes sense, but this one will be any connection after we unite the list

Reply:
yea that is true but that I would prefer to handle in the POC branch. Since this will land sooner than the POC I was keeping the context as it is.